cli/cli/Commands/Project/GenerateEnvFileCommand.cs

```diff
@@ -36,30 +41,26 @@ public override void Configure()
 		AddOption(new Option<bool>("--auto-deploy", () => false, "When enabled, automatically deploy dependencies that aren't running"), (args, i) => args.autoDeploy = i);
 	}
 
-	public override async Task Handle(GenerateEnvFileCommandArgs args)
+	public override async Task<GenerateEnvFileOutput> GetResult(GenerateEnvFileCommandArgs args)
 	{
 		var realmsApi = args.Provider.GetService<IRealmsApi>();
 		var res = await realmsApi.GetAdminCustomer();
 		var proj = res.customer.projects.FirstOrDefault(p => p.name == args.AppContext.Pid);
 
-		var secret = proj.secret;
-		var cid = args.AppContext.Cid;
-		var pid = args.AppContext.Pid;
-		var prefix = args.includePrefix ? MachineHelper.GetUniqueDeviceId() : "";
-		var host = args.AppContext.Host.Replace("http", "ws") + "/socket";
-		var refreshToken = args.AppContext.RefreshToken;
-
-
-		var fileContent = @$"SECRET={secret}
-HOST={host}
-CID={cid}
-PID={pid}
-NAME_PREFIX={prefix}
-BEAM_INSTANCE_COUNT={args.instanceCount}
-REFRESH_TOKEN={refreshToken}
-";
-
-
+		var output = new GenerateEnvFileOutput
+		{
+			envVars = new List<EnvVarOutput>
+			{
+				EnvVarOutput.Create("SECRET", proj.secret),

```

Comment:
Is there any possibility for these environment variables conflict with different ones in that process we are adding them? If so, would be nice to add a prefix or something to these.

Reply:
fair enough feedback, but I think that would amount to a larger breaking change than we should do in a patch release. Do you mind if I make a ticket that gets on a 2.0 milestone?

Reply:
seems good enough

Reply:
https://github.com/beamable/BeamableProduct/issues/3231